src/staging/src/components/park/about.js

```diff
@@ -1,5 +1,9 @@
 import React from "react"
+<<<<<<< Updated upstream
 import { parseISO, format } from "date-fns"
+=======
+import { capitalize } from "lodash"
+>>>>>>> Stashed changes
```

Comment:
Looks like a merge that didn't get resolved here

Reply:
Oops! Good catch! Have pushed the fix.